cilium-etcd-cluster/descriptors.go

```diff
@@ -61,6 +61,17 @@ func CiliumEtcdCluster(namespace, repository, version string, size int, etcdEnv
 					},
 				},
 				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								MatchExpressions: []v1.NodeSelectorRequirement{
```

Comment:
Should we go ahead and add `cloud.google.com/gke-preemptible != "true"` as well?

Reply:
ohhh, yes!